test/transform/test_generic.py

```diff
@@ -34,6 +34,18 @@ def _test_tile_impl(ndim, reps):
         actual = tile_aug(a)
         assert actual.shape == expected.shape
 
+    def test_resize_longest_inputs(self):
+        input_shape = (520, 704)
+        target_shape = (512, 512)
```

Comment:
I would rather test this for a few different input and target shapes.
E.g.
```python
input_shapes = [(520, 704), (256, 384), ...]
target_shapes =  [(512, 512), (1024, 1024), ...]
```
And then have a loop that runs the test over those. And then check different cases, like target shape smaller and bigger than input shape.

Reply:
Added more combinations for it!